tensorflow/python/keras/layers/normalization.py

```diff
@@ -613,8 +613,50 @@ def _fake_update():
 
     return (r, d, out_mean, out_variance)
 
-  def _moments(self, inputs, reduction_axes, keep_dims):
-    mean, variance = nn.moments(inputs, reduction_axes, keep_dims=keep_dims)
+  def moment_calculation(self,
```

Comment:
Can you make this a private method (_underscore in the name) so it doesn't become a part of TF's public API?